SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialConnectionManager.cpp

```diff
@@ -468,6 +468,12 @@ void USpatialConnectionManager::SetupConnectionConfigFromURL(const FURL& URL, co
 			FParse::Value(FCommandLine::Get(), TEXT("locatorHost"), LocatorHostOverride);
 		}
 
+		int32 LocatorPortOverride = SpatialConstants::LOCATOR_PORT;
+		if (const TCHAR* LocatorPortOption = URL.GetOption(TEXT("customPort="), nullptr))
```

Comment:
Hmmm, since this explicitly only overrides the `LocatorPort` and not the port used for Receptionist, should we call the URL option `customLocatorPort` or perhaps just `locatorPort`?